### ISSUE TITLE
AMD: Object #<Object> has no method 'buildeExecute' 

### ISSUE THREAD

**Issue by Munter** [systemjs/builder#275]
I'm running jspm 0.16.0-beta7 and systemjs-builder 0.13.1 and am trying to compile underscore templates using https://github.com/podio/plugin-jst/blob/master/jst.js and underscore as an amd module.

This is my code:

```js
define(function (require) {
  return require('path/to/template!jst');
});
```

When building this file it throws `Object #<Object> has no method 'builderExecute' `

**Comment by guybedford** [comment]
@Munter can you try searching your files to see where this `buildeExecute` is coming from? I can't seem to see it anywhere in my code, but I'm guessing this is a version mismatch of some sorts?

**Comment by Munter** [comment]
Woops, typo. This is where it looks like it comes from: https://github.com/systemjs/systemjs/blob/4524c2e71eea0f8eb35ee786aacfa91bf778181a/lib/amd.js#L71

**Comment by guybedford** [comment]
@Munter ah right, thought it was my typo.

Ok I just tried the following:
* jspm install jst=github:podio/plugin-jst underscore
* creating a dummy template and copying your define
* Opening that in the browser works fine
* Bundling `jspm bundle main` loading the template also works fine

Any tips to reproduce further?

**Comment by matthewbauer** [comment]
I've encountered that same issue. But, I can't remember how/if I fixed it. I think it had something to do with version mismatch of global vs local jspm. I can try to reproduce it later.

**Comment by guybedford** [comment]
Yes I think it may have been an older bug. Let me know if you still get this with the latest updates.

**Comment by guybedford** [comment]
Ok yes this was a bug with AMD plugins. I've added a fix in https://github.com/systemjs/builder/commit/d2e6c3397821e8d41c9e7d33c756704f1548f69d.

**Comment by guybedford** [comment]
Released in 0.13.4.
